### PR TITLE
Update metadata only handling

### PR DESCRIPTION
**Pull request by sephirothkod**

### CODE REVIEW

File: app/controllers/bulkrax/importers_controller.rb [hunk 1]
```diff
@@ -115,21 +115,9 @@ def update
...
-          @importer.parser_fields['update_files'] = true
-          @importer.save
-          Bulkrax::ImporterJob.perform_later(@importer.id)
-          # Perform a full metadata and files re-import; do the same for an OAI re-harvest of all items
-        elsif params[:commit] == ('Update and Replace Files' || 'Update and Re-Harvest All Items')
-          @importer.parser_fields['replace_files'] = true
-          @importer.save
-          Bulkrax::ImporterJob.perform_later(@importer.id)
-        # In all other cases, perform a metadata-only re-import
         else
```
Comment (alishaevn) [inline app/controllers/bulkrax/importers_controller.rb:118]:
```suggestion
        unless params[:commit] == 'Update Importer'
```

File: app/controllers/bulkrax/importers_controller.rb [hunk 2]
```diff
@@ -115,21 +115,9 @@ def update
         # do not perform the import
         if params[:commit] == 'Update Importer'
         # do nothing
```
Comment (alishaevn) [inline app/controllers/bulkrax/importers_controller.rb:117]:
```suggestion
```

File: app/parsers/bulkrax/csv_parser.rb [hunk 3]
```diff
@@ -127,8 +127,13 @@ def current_work_ids
...
         complete_entry_identifiers = complete_statuses.map { |s| s.statusable&.identifier&.gsub(':', '\:') }
-
-        ActiveFedora::SolrService.query("#{work_identifier}_tesim:(#{complete_entry_identifiers.join(' OR ')})#{extra_filters}", rows: 2_000_000_000).map(&:id)
+        extra_filters = extra_filters.presence || '*:*'
+        ActiveFedora::SolrService.get(
+            extra_filters.to_s,
+            fq: "#{work_identifier}_sim:(#{complete_entry_identifiers.join(' OR ')})",
+            fl: 'id',
+            rows: 2_000_000_000
+        )['response']['docs'].map{ |obj| obj['id'] }
```
Comment (alishaevn) [inline app/parsers/bulkrax/csv_parser.rb:136]:
```suggestion
        extra_filters = extra_filters.presence || '*:*'

        ActiveFedora::SolrService.get(
            extra_filters.to_s,
            fq: "#{work_identifier}_sim:(#{complete_entry_identifiers.join(' OR ')})",
            fl: 'id',
            rows: 2_000_000_000
          )['response']['docs'].map { |obj| obj['id'] }
```

Comment (alishaevn) [reply app/parsers/bulkrax/csv_parser.rb:136]: fixes for the rubocop errors. 🙂

**Comment by alishaevn** [comment]
when testing this pr locally I'm able to once again export works that have files attached to them. 🎉

**Review by alishaevn — APPROVED** [review]

**Review by orangewolf — APPROVED** [review]

**Pull request merged**